docs/data_management/tutorial/globus_individual_tutorial.md

```diff
@@ -0,0 +1,367 @@
+
+## How Do I Find UAB Storage Collections?
+
+UAB offers [multiple storage resources](../storage.md). The following resources may be accessed through Globus Collections.
+
+- Cheaha file system (GPFS) including [individual directories](../storage.md#what-individual-storage-solutions-are-available) and [project directories](../storage.md#project-directory).
+- [Long-Term Storage (LTS)](../lts/index.md).
+- [UAB Box](https://www.uab.edu/it/home/tech-solutions/file-storage/box).
+
+To find these resources on Globus, use the following steps.
+
+1. [Get onto the Globus Web App](#how-do-i-get-onto-the-globus-web-app).
+1. You should see the "File Manager" page. If not click "File Manager" in the left-hand navigation panel.
+
+    ![File Manager page.](./images/common/file-manager.png)
+
+1. Click the "Collection Search" bar to open the "Collection Search" page.
+
+    ![Collection search page](./images/common/collection-search-page.png)
+
+1. In the search bar, type one of the following, depending on which resource you need, and select the appropriate entry.
+
+    - **Cheaha Filesystem (GPFS)**: Type "UAB Cheaha". There are two Collections, choose one based on where the other computer is located.
+        - **(1)** Transferring with a computer on the UAB Campus Network or UAB Wifi? Select "Cheaha cluster **on**-campus (UAB Science DMZ)".
+        - **(2)** Transferring with a computer on other networks? select "Cheaha cluster **off**-campus (UAB Science DMZ)".
+
+        ![UAB Box search results.](./images/gi-uab-collections/001-cheaha.png)
+
+    - **Long-Term Storage (LTS)**: Type "UAB LTS" and select the entry labeled "UAB Research Computing LTS (Long Term Storage aka S3)".
+
+        ![UAB Box search results.](./images/gi-uab-collections/002-lts.png)
+
+    - **UAB Box**: Type "UAB Box" and select the entry labeled "UAB Box".
+
+        ![UAB Box search results.](./images/gi-uab-collections/003-box.png)
+
+Proceed on to learn [how to find Collections shared with you](#how-do-i-find-uab-storage-collections).
+
+[Return to the top](#globus-tutorials-for-individual-researchers).
+
+## How Do I Find Collections Shared with Me?
+
+Globus allows users to share Collections with others. Other researchers, labs, and Research Cores on campus may invite you to their Collections to share data with you. The following instructions let you view Collections shared with you.
+
+1. [Log in to Globus using your credentials.](#how-do-i-get-onto-the-globus-web-app)
+1. You should see the "File Manager" page. If not click "File Manager" in the left-hand navigation panel.
+
+    ![File Manager page.](./images/common/file-manager.png)
+
+1. Click the "Collection Search" bar to open the "Collection Search" page.
+
+    ![Collection search page](./images/common/collection-search-page.png)
+
+1. The list of Collections will be filtered down to all Collections others have granted you access to, which should help you find the Collections you need.
```

Comment:
In this step, I would suggest adding text related to the selection of **Shared With You** tab since the default is the **Recent** tab. May be something like, "1. Select the "Shared With You" tab. The list of Collections will be filtered to show only those that others have granted you access to..."

Reply:
Done, added to line 186.